src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabUtilization/components/NetworkUtil/NetworkUtil.tsx

```diff
@@ -20,38 +25,92 @@ const NetworkUtil: React.FC<NetworkUtilProps> = ({ vmi }) => {
   const [networkIn] = usePrometheusPoll({
     query: queries?.NETWORK_IN_USAGE,
     endpoint: PrometheusEndpoint?.QUERY,
     namespace: vmi?.metadata?.namespace,
     endTime: currentTime,
   });
 
+  const [networkTotal] = usePrometheusPoll({
+    query: queries?.NETWORK_TOTAL_BY_INTERFACE_USAGE,
+    endpoint: PrometheusEndpoint?.QUERY,
+    namespace: vmi?.metadata?.namespace,
+    endTime: currentTime,
+  });
+
   const [networkOut] = usePrometheusPoll({
     query: queries?.NETWORK_OUT_USAGE,
     endpoint: PrometheusEndpoint?.QUERY,
     namespace: vmi?.metadata?.namespace,
     endTime: currentTime,
   });
-
+  const networkInterfaceTotal = Number(networkTotal?.data?.result?.[0]?.value?.[1]).toFixed(2);
   const networkInData = +networkIn?.data?.result?.[0]?.value?.[1];
   const networkOutData = +networkOut?.data?.result?.[0]?.value?.[1];
-
   const totalTransferred = xbytes(networkInData + networkOutData || 0, {
     iec: true,
     fixed: 0,
   });
-
+  const networkInterfacesLength = interfacesNames?.length;
   const isReady = !isEmpty(networkInData) || !isEmpty(networkOutData);
 
   return (
     <div className="util network">
       <div className="util-upper">
         <div className="util-title">
           {t('Network Transfer')}
-          <div className="util-title__subtitle text-muted">{t('Primary Network')}</div>
+          <Popover
+            bodyContent={
+              <div>
+                <Text component={TextVariants.h3}>{t('Network transfer breakdown')}</Text>
+                <Text component={TextVariants.h6}>{t('Top consumer')}</Text>
+                {interfacesNames?.map((networkInterface) => (
+                  <div key={networkInterface.name} className="network-poppver">
+                    <Link
+                      to={`${getResourceUrl({
+                        model: VirtualMachineModel,
+                        resource: vmi,
+                      })}/metrics?network=${networkInterface.name}`}
+                    >
+                      {networkInterface.name}
+                    </Link>
+                    {networkInterface.name === networkInterfaceTotal ? (
+                      <div className="text-muted">{`${networkInterfaceTotal} MBps`}</div>
+                    ) : (
+                      <div className="text-muted">
+                        {networkTotal?.data?.result?.map(
+                          (name) =>
+                            name?.metric?.interface === networkInterface.name &&
+                            `${Number(name?.value?.[1]).toFixed(2)} MBps`,
+                        )}
+                      </div>
+                    )}
+                  </div>
+                ))}
+                {networkInterfacesLength > 5 && (
+                  <Link
+                    to={`${getResourceUrl({
+                      model: VirtualMachineModel,
+                      resource: vmi,
+                    })}/metrics?network=All networks`}
```

Comment:
This looks kinda weird, I mean the space in the link, between `All` and `networks`. Are you sure this is working as expected?

Reply:
Yes, it works as expected, but I deleted "all networks" and it will move to the same place I wanted,
so it's better to delete, thank you :)